lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  attr_reader :list
+  def initialize
+    @list = [
+      {:name => "panini", :cost => "2"},
+      {:name => "biriyani", :cost => "3"},
+      {:name => "cake", :cost => "4"}
+    ]
+  end
+
+  # def select_dishes(item)
```

Comment:
I'm guessing this method was removed from this class and then moved up to the Order class?

If that's the case I would recommend deleting the code here rather than just commenting it out - it's an easy way of keeping your code looking neat and tidy.